src/main/java/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java

```diff
@@ -188,21 +187,21 @@ private static void findScrambleOptions(final PlayerID player, final Map<Territo
     }
     final Match<Unit> airbasesCanScramble = Match.all(Matches.unitIsEnemyOf(data, player),
         Matches.UnitIsAirBase, Matches.UnitIsNotDisabled, Matches.unitIsBeingTransported().invert());
-    final CompositeMatchAnd<Territory> canScramble = new CompositeMatchAnd<>(
-        Match.any(Matches.TerritoryIsWater, Matches.isTerritoryEnemy(player, data)),
-        Matches.territoryHasUnitsThatMatch(Match.all(Matches.UnitCanScramble,
-            Matches.unitIsEnemyOf(data, player), Matches.UnitIsNotDisabled)),
-        Matches.territoryHasUnitsThatMatch(airbasesCanScramble));
+    final Match.CompositeBuilder<Territory> canScrambleBuilder = Match.<Territory>newCompositeBuilder()
+        .add(Match.any(Matches.TerritoryIsWater, Matches.isTerritoryEnemy(player, data)))
+        .add(Matches.territoryHasUnitsThatMatch(Match.all(Matches.UnitCanScramble,
+            Matches.unitIsEnemyOf(data, player), Matches.UnitIsNotDisabled)))
+        .add(Matches.territoryHasUnitsThatMatch(airbasesCanScramble));
```

Comment:
Agreed, and I'd have happily reformatted things to make it easier to read.  However, I was thinking about the reviewer's sanity in this set of changes. :smile:  I was trying to keep formatting changes to a minimum so, as you can see, the GitHub diff viewer only highlights the characters that changed instead of the entire line.

This may be too verbose, but I probably would have reformatted the above statement as follows:

```java
final Match.CompositeBuilder<Territory> canScrambleBuilder = Match.<Territory>newCompositeBuilder()
    .add(Match.any(
        Matches.TerritoryIsWater,
        Matches.isTerritoryEnemy(player, data)))
    .add(Matches.territoryHasUnitsThatMatch(Match.all(
        Matches.UnitCanScramble,
        Matches.unitIsEnemyOf(data, player),
        Matches.UnitIsNotDisabled)))
    .add(Matches.territoryHasUnitsThatMatch(airbasesCanScramble));
```